tests/AnalyzeSpec.hs

```diff
@@ -778,19 +778,39 @@ spec = describe "analyze" $ do
     expectPass code $ Satisfiable Abort'
     expectPass code $ Satisfiable Success'
 
-  describe "create-user-guard" $ do
+  describe "create-user-guard passing" $ do
     let code =
           [text|
-            (defun fail:bool (o:object{account})
+            (defun enforce-range:bool (x:integer)
+              (enforce (> x 1) ""))
+
+            (defun test:bool (x:integer)
+              @model [(property (> x 1))]
+              (enforce-guard (create-user-guard (enforce-range x))))
+          |]
+    expectVerified code
+
+  describe "create-user-guard failing" $ do
+    let code =
+          [text|
+            (defun enforce-impossible:bool ()
               (enforce false ""))
 
-            (defun test:bool ()
-              (enforce-guard (create-user-guard {} "fail")))
+            (defun test:bool (x:integer)
```

Comment:
Why'd you add `x` here?

Reply:
I guess it probably wasn't conceived of that way, though github's diff shows an added arg.